REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.2-
-URIParser
-SHA
\ No newline at end of file
+URIParser 0.0.5
```

Comment:
As I commented in #137, I don't think we should use the REQUIRE file just to eliminate warnings.  We should only list a version number if that version is required for BinDeps to function.